Applications/Spire/Source/KeyBindings/TagModel.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Spire/KeyBindings/TagModel.hpp"
+#include <algorithm>
+
+using namespace Spire;
+
+TagModel::TagModel(std::vector<Schema> schemas)
+  : m_schemas(std::move(schemas)) {}
+
+std::string TagModel::get_tag_name(int tag_key) const {
+  return std::find_if(m_schemas.begin(), m_schemas.end(), [=] (auto& schema) {
```

Comment:
Capture by reference.

Reply:
So, capture by reference even applies to captures less than a pointer size, right?

Reply:
Good observation but yes, the choice of capture will be based on lifetime rather than optimization and we'll defer to the compiler to optimize it away through inlining.